go/pkg/pass1/set-path.go

```diff
@@ -55,29 +106,24 @@ func setpathObj(obj pathObj, intfToZone1 *routerIntf, distToZone1 int) (int, *lo
 	obj.setDistance(distToZone1)
 	maxDistance := distToZone1
 
+	r, isRouter := obj.(*router)
```

Comment:
Würde ich in eine Zeile zusammenziehen:
if r, isRouter := obj.(*router); isRouter {
oder kürzer
if r, ok := obj.(*router); ok {

Reply:
OK, isRouter  wird weiter unten verwendet :-)